test/fixtures/options.anchor/anchor-presets-fallback-matrix.js

```diff
@@ -0,0 +1,61 @@
+var datasets = [];
+
+var data = [
+  {x: 1, y: 1, v: 11},
+  {x: 1, y: 2, v: 12},
+  {x: 2, y: 1, v: 21},
+  {x: 2, y: 2, v: 22},
+];
+
+['start', 'center', 'end'].forEach(function(anchor) {
+  datasets.push({
+    data: data,
+    datalabels: {
+      anchor: anchor
+    },
+    width: 128,
+    height: 96
+  });
+});
+
+export default {
+  config: {
+    type: 'matrix',
```

Comment:
While not a blocker, I'm not sure I like this test against an external plugin since it may fail not because of the datalabels logic but the matrix one. What would be required to remove this "matrix" dependency and write a very minimal generic test that doesn't depend on the logic of an external plugin?

Reply:
Created a test controller/element for both cases, with and without width/height

Reply:
That's awesome! I didn't think it would be that simple.